dotCMS/src/main/java/com/dotcms/vanityurl/model/CachedVanityUrl.java

```diff
@@ -0,0 +1,126 @@
+package com.dotcms.vanityurl.model;
+
+import com.dotmarketing.util.Logger;
+import com.liferay.util.StringPool;
+import com.liferay.util.StringUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.liferay.util.StringUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)